core/src/components/loading/test/a11y/loading.e2e.ts

```diff
@@ -2,27 +2,48 @@ import AxeBuilder from '@axe-core/playwright';
 import { expect } from '@playwright/test';
 import { configs, test } from '@utils/test/playwright';
 
-configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
+configs({ modes: ['ios'], directions: ['ltr'], themes: ['light', 'dark'] }).forEach(({ title, config }) => {
```

Comment:
Optional: We don't use the `index.html` file in this test anymore. Should we remove it?